xapian-core/net/remoteprotocol.h

```diff
@@ -132,6 +133,7 @@ enum reply_type {
     REPLY_METADATAKEYLIST,	// Iterator for metadata keys
     REPLY_FREQS,		// Get termfreq and collfreq
     REPLY_UNIQUETERMS,		// Get number of unique terms in doc
+    REPLY_WDFDOCMAX,		// Get the max_wdf in doc
```

Comment:
`remote_protocol.rst` needs updating with the details of these new messages.

Reply:
Updated.